dataland-frontend/src/utils/api/CompanyDataOwner.ts

```diff
@@ -0,0 +1,34 @@
+import type Keycloak from "keycloak-js";
+import { ApiClientProvider } from "@/services/ApiClients";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import { isCompanyIdValid } from "@/utils/ValidationsUtils";
+
+/**
+ * Get the Information about Data-ownership
+ * @param getKeycloakPromise getter for a keycloak promise
+ * @param companyId identifier of the company
+ * @returns a promise which resolves to a boolean if the company has at least one data owner
+ */
+export async function getCompanyDataOwnerInformation(
+  getKeycloakPromise: () => Promise<Keycloak>,
```

Comment:
Ich glaube hier kann man ein bisschen streamlinen. Schaut euch doch mal die Funktion "isUserDataOwnerForCompany" in DataOwnerUtils an.
Da könnt ihr die Variable atLeastOneDataOwner sowie die ganzen assertDefined einsparen.

Reply:
guter Punkt - habe es angepasst

Reply:
ok